packages/cf-component-pagination/src/PaginationRoot.js

```diff
@@ -0,0 +1,17 @@
+import { createComponent } from 'cf-style-container';
+import PaginationRootTheme from './PaginationRootTheme';
+import { applyTheme } from 'cf-style-container';
+
+const paginationRootStyles = ({ theme }) => ({
+  '&:after': {
+    content: theme.content,
+    display: theme.display,
+    clear: theme.clear
+  }
+});
+
+const PaginationRootUnstyled = createComponent(paginationRootStyles, 'div');
+
+export { PaginationRootUnstyled };
+
+export default applyTheme(PaginationRootUnstyled, PaginationRootTheme);
```

Comment:
I use `export default applyTheme(PaginationRootUnstyled, PaginationRootTheme)` to import into Pagination, I don't want duplication in index.js and PaginationRoot